test/Maestro/EU/EU_delete_account.yml

```
@@ -0,0 +1,28 @@
+appId: net.givtapp.ios.test
+name: When logged in this is how you delete an user
+
+env:
+    PASSWORD: Test123
+---
+- tapOn:
+    label: "Click on the menu icon"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)